### PR TITLE
Added an watsonx and langchain integratioin example

### PR DESCRIPTION
**Pull request by gyliu513**
- [ ] I have added tests that cover my changes.
- [ ] If adding a new instrumentation or changing an existing one, I've added screenshots from some observability platform showing the change.
- [ ] PR name follows conventional commits format: `feat(instrumentation): ...` or `fix(instrumentation): ...`.
- [ ] (If applicable) I have updated the documentation accordingly.

### CODE REVIEW

**Comment by gyliu513** [comment]
@nirga can you check the CI error? Seems not caused by my code, thanks!

**Comment by nirga** [comment]
weird @gyliu513 
looks like a transient error. Retrying

File: packages/sample-app/sample_app/watsonx-langchain.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+question_simple = "What is 1 + 1?"
+
+response = watsonx_llm_generate(question_simple)
+
+if isinstance(response, types.GeneratorType):
+    for chunk in response:
+        print(chunk, end='')
+
+pprint(response)
```
Comment (nirga) [inline packages/sample-app/sample_app/watsonx-langchain.py:54]:
```suggestion
print(response)
```

**Review by nirga — APPROVED** [review]
@gyliu513 great, working! can you resolve the lint issue?

**Pull request merged**